content/doc/book/pipeline/index.adoc

```diff
@@ -120,6 +143,76 @@ modeled in Jenkins Pipeline:
 
 image::pipeline/realworld-pipeline-flow.png[title="Pipeline Flow", 800]
 
+[[jenkinsfile]]
+== The Jenkinsfile
+
+A `Jenkinsfile` is a text file that contains the definition of a
+Jenkins Pipeline and is checked into source control.
+footnoteref:[scm, https://en.wikipedia.org/wiki/Source_control_management]
+This is the foundation of "Pipeline-as-Code"; treating the continuous delivery
+pipeline a part of the application to be version and reviewed like any other code.
```

Comment:
missing "as" and an "ed" on version for the sake of parallel structure (should read: "treating the continuous delivery pipeline as a part of the application to be versioned and reviewed...")